### DISCUSSION TITLE
What do you think about this responsive table

### DISCUSSION THREAD

**Discussion by sovetski** [saadeghi/daisyui#3033] [Ideas / feature requests]
Hi,

Tables are always a problem on small screens, even if we have a scrollbar, the user experience is not very good.

What do you think about this idea? https://codepen.io/AllThingsSmitty/pen/MyqmdM

If you check the code, we have only 1 code and it is a basic HTML Table structure

It is a simple table on large screens:
![image](https://github.com/saadeghi/daisyui/assets/13520683/cc7092f9-185c-4640-8201-49261143f92f)

And on mobile:
![image](https://github.com/saadeghi/daisyui/assets/13520683/dafcb510-14a9-4b66-bf4d-ca6a55ae5270)

If it is ok for everyone I can try to do a PR for this

**Comment by TheLastCookie-404** [comment]
Looks great
